content/en/blog/postman-grpcweb.md

```diff
@@ -0,0 +1,328 @@
+---
+title: Can gRPC replace REST and WebSockets for Web Application Communication?
+date: 2023-12-04
+author:
+  name: Ian Douglas
+  position: Sr Developer Advocate, Postman
+  link: https://linkedin.com/in/iandouglas736
+---
+
+{{% alert color="info" %}}
+Welcome to this guest blog entry from our friends at
+[Postman](https://www.postman.com/)!
+{{% /alert %}}
+
+In the rapidly evolving landscape of web development, efficiency and
+performance often stand at the forefront of adoption of new technologies.
+The work being done with the [gRPC-Web](https://github.com/grpc/grpc-web)
+library marks a pivotal shift in how developers can utilize the speed and
+power of [gRPC](https://grpc.io/) into client-server communication in web
+applications by replacing REST and some aspects of WebSockets. Let’s look at a
+comparative analysis with traditional RESTful calls and WebSocket connections,
+and offer practical code samples with gRPC-Web to draw comparisons in each approach.
+
+
+## Understanding gRPC-Web and Its Place in Modern Web Development
+
+The genesis of gRPC-Web lies in the quest for more responsive, low-latency web
+applications. It extends the capabilities of gRPC – a high-performance,
+open-source universal RPC framework – to the browser, enabling direct
+communication with gRPC services normally designated to server-to-server
+communication. gRPC is structured around a serialization format called [Protocol
+Buffers (Protobuf)](https://protobuf.dev/), which facilitates smaller payloads
+and well-defined interface descriptions that streamline the development process.
+
+Before diving into the technicalities, let’s examine the potential shift that
+gRPC-Web represents. Unlike REST, which uses HTTP/1.1 for stateless server
```

Comment:
It's also not true that REST uses HTTP/1.1, because the protocol is negotiated by the browser, and nowadays HTTP/2 is preferred most of the time.

Let's follow up in the google doc and address is before publishing. Thanks!

Reply:
I recommend to rewrite the sentence starting with "Unlike REST" with the following:

_Unlike the native gRPC protocol which requires HTTP/2, gRPC-Web relaxed this requirement so it can support any HTTP/* protocols available in a browser environment._

Reply:
Thanks, I have incorporated that change.